src/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -50,78 +50,66 @@ export const DiscoveryTroubleshootingModal: React.FC<DiscoveryTroubleshootingMod
     >
       <TextContent>
         <Text component={TextVariants.p}>
-          Booting from an ISO takes a few minutes. It also depends heavily on the hardware and
-          network configuration. However, if you have verified that your host machine is powered on,
-          DHCP is enabled and yet the host does not appear in the inventory list, then login to the
-          host over SSH to troubleshoot. For that, you would need to know your machine IP-address.
-          You can also use a console such as BMC or virtual machine console.
+          To troubleshoot host discovery issues, complete the following steps:
         </Text>
+        <Text component={TextVariants.h2}>Verify that your host machine is powered on</Text>
+        <Text component={TextVariants.p}>
+          Note: Boot time depends on several factors such as your hardware and network
+          configuration, and if you are booting from an ISO.
+        </Text>
+        <Text component={TextVariants.h2}>Verify if DHCP is enabled</Text>
         <Text component={TextVariants.h2}>SSH into your machine</Text>
-        <PrismCode code={`ssh core@<machine-ip>`} />
         <Text component={TextVariants.p}>
-          The SSH key provided when generating the discovery ISO is used to authenticate the user.
-          It should not be required to provide the password.
+          Verify that you can access your host machine using SSH, or a console such as BMC or
+          virtual machine console. In the CLI, enter the following command:
+        </Text>
+        <PrismCode code="ssh core@<machine-ip>" />
+        <Text component={TextVariants.p}>
+          NOTE: Authentication is provided by the discovery ISO, therefore when you access your host
```

Comment:
This works just in case that default private key on the machine/session of current user matches the public key provided when generating the ISO.
I would mention the `-i` param here. Or at least note about keys (which one is actually used).

Reply:
What do you suggest?

Reply:
To extend the hint to cover even the case when the default key is not matching the key provided in the ISO discovery modal.

If the existing "happy-path" fails, the user needs to use `ssh -i [MATCHING_PRIVATE_KEY]` to access the host.

Reasons for an issue include:
- different machine
- different user
- multiple keys used by the user
- a key previously generated just for the cluster installation

Reply:
@olivelakra Can you please have a look ^^^ ?

Reply:
So we can replace `ssh core@<machine-ip>` by ssh -i [MATCHING_PRIVATE_KEY] which is more generic.